adapters/audienceNetwork/audienceNetworktest/exemplary/video_consented_providers.json

```diff
@@ -0,0 +1,147 @@
+                    "mimes": ["video/mp4"],
+                    "minduration": 15,
+                    "maxduration": 30,
+                    "protocols": [2, 3, 5, 6, 7, 8],
+                    "linearity": 1,
+                    "w": 940,
+                    "h": 560
+                },
+                "ext": {
+                    "bidder": {
+                        "publisherid": "123",
+                        "placementid": "456"
+                    }
+                }
+            }
+        ],
+        "app": {
+            "id": "app-abc",
+            "bundle": "com.prebid"
+        },
+        "device": {
+            "ip": "152.193.6.74"
+        },
+        "user": {
+            "id": "db089de9-a62e-4861-a881-0ff15e052516",
+            "buyeruid": "v4_bidder_token",
+            "ext": {
+                "consent": "TESTCONSENT",
+                "consented_providers_settings": {
+                    "consented_providers": [1608,765,492,1365,5678,1545,2563,1411]
+                },
+                "test": 123
+            }
+        },
+        "tmax": 500
+    },
+    "httpcalls": [
+        {
+            "expectedRequest": {
+                "uri": "https://an.facebook.com/placementbid.ortb",
+                "headers": {
+                    "Accept": [
+                        "application/json"
+                    ],
+                    "Content-Type": [
+                        "application/json;charset=utf-8"
+                    ],
+                    "X-Fb-Pool-Routing-Token": [
+                        "v4_bidder_token"
+                    ]
+                },
+                "body": {
+                    "id": "test-imp-id",
+                    "imp": [
+                        {
+                            "id": "test-imp-id",
+                            "video": {
+                                "mimes": ["video/mp4"],
+                                "minduration": 15,
+                                "maxduration": 30,
+                                "protocols": [2, 3, 5, 6, 7, 8],
+                                "linearity": 1,
+                                "w": 0,
+                                "h": 0
+                            },
+                            "tagid": "123_456"
+                        }
+                    ],
+                    "app": {
+                        "id": "app-abc",
+                        "bundle": "com.prebid",
+                        "publisher": {
+                            "id": "123"
+                        }
+                    },
+                    "device": {
+                        "ip": "152.193.6.74"
+                    },
+                    "user": {
+                        "id": "db089de9-a62e-4861-a881-0ff15e052516",
+                        "buyeruid": "v4_bidder_token",
+                        "ext": {
+                            "consent": "TESTCONSENT",
+                            "test": 123
```

Comment:
`"test": 123` is not a valid part of the Facebook request. I recommend excluding it from this test and relying on the unit tests to ensure other json elements aren't inappropriately affected.

Reply:
Removed